app/org/sagebionetworks/bridge/models/activities/CustomActivityEventRequest.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.models.activities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Created by jyliu on 6/12/2017.
+ */
+public class CustomActivityEventRequest {
+    private final String eventKey;
+    private final Long timestamp;
+
+    @JsonCreator
+    public CustomActivityEventRequest(@JsonProperty("eventKey") String eventKey,
+            @JsonProperty("timestamp") Long timestamp) {
+        this.eventKey = eventKey;
+        this.timestamp = timestamp;
+    }
+
+
+    public String getEventKey() {
+        return eventKey;
+    }
+
+    public Long getTimestamp() {
+        return timestamp;
```

Comment:
This should be a DateTime value, that serializes to an ISO 8601 time, like all other instances in API.